documentation/CHANGELOG_TEMPLATE.md

```diff
@@ -37,4 +48,4 @@ The below sections record the changes for each API version in each release as fo
 
 ### Removed
 
-<!--In case of a release with multiple APIs repeat the above API changes sections and its sub-sections for each API changed in this release -->
+<!--In case of a release with multiple APIs: repease the above section (level 2 and 3 headers) to hold the changes for each API version in this release-->
```

Comment:
```suggestion
<!--In case of a repository with multiple APIs: repeat the above section (level 2 and 3 headers) to hold the changes for each API version in this release, also mention if an API within the repository is unchanged-->
```

Reply:
Update as follows: add an annotation "- unchanged" in the API list, and then add sections for the APIs that have changed.